client/src/components/Login.js

```diff
@@ -57,18 +57,57 @@ signupLink:{
 }));
 function Login (){
     const classes = useStyles();
-    const [userEmail,setEmail]=useState('')
-    const [userPass, setPass]=useState('')
+    const [email,setEmail]=useState('');
+    const [password, setPassword]=useState('');
+    const [errors, setErrors]=useState('');
```

Comment:
If this eventually will be an object, we want to start it as an object - `=useState({})`